coprarna_aux/homology_intaRNA.pl

```diff
@@ -298,106 +297,90 @@
+    system "cp " . $PATH_COPRA_SUBSCRIPTS . "CopraRNA_available_organisms.txt ."; 
+    system "R --slave -f " . $PATH_COPRA_SUBSCRIPTS . "copraRNA2_phylogenetic_sorting.r 2> $OUT_ERR > $OUT_STD"; 
     # perform actual CopraRNA 2 p-value combination
-    system "R --slave -f " . $PATH_COPRA_SUBSCRIPTS . "join_pvals_coprarna2.R --args $ooi_refseq_id ooi_consensus overall_consensus 2> /dev/null > /dev/null"; ## edit 2.0.6
+    system "R --slave -f " . $PATH_COPRA_SUBSCRIPTS . "join_pvals_coprarna_2.r 2> $OUT_ERR > $OUT_STD"; 
     
 }
 
-# truncate final output // ## edit 2.0.5.1
-system "head -n $topcount CopraRNA1_final_all.csv > CopraRNA1_final.csv" if ($cop1); ## edit 2.0.6
+#edit jens
+# truncate final output // 
+system "head -n $topcount CopraRNA1_final_all.csv > CopraRNA1_final.csv" if ($cop1); 
 unless ($cop1) {
-    system "head -n $topcount CopraRNA2_final_all_ooi.csv > CopraRNA2_final_ooi.csv"; ## edit 2.0.6
-    system "head -n $topcount CopraRNA2_final_all_balanced.csv > CopraRNA2_final_balanced.csv"; ## edit 2.0.6
-    system "head -n $topcount CopraRNA2_final_all_balanced_consensus.csv > CopraRNA2_final_balanced_consensus.csv"; ## edit 2.0.6
-    system "head -n $topcount CopraRNA2_final_all_ooi_consensus.csv > CopraRNA2_final_ooi_consensus.csv"; ## edit 2.0.6
-    system "head -n $topcount CopraRNA2_final_all_ooi_ooiconsensus.csv > CopraRNA2_final_ooi_ooiconsensus.csv"; ## edit 2.0.6
-}
-
-# figure out which result is the primary result ## edit 2.0.6
-if ($cop1) { # CopraRNA 1 is the primary requested result
-    system "cp CopraRNA1_final.csv CopraRNA_result.csv";    
-    system "cp CopraRNA1_final_all.csv CopraRNA_result_all.csv";    
-} elsif ($nooi and (not $cons)) { # CopraRNA 2 with balanced mode is the requested result
-    system "cp  CopraRNA2_final_balanced.csv CopraRNA_result.csv";
-    system "cp  CopraRNA2_final_all_balanced.csv CopraRNA_result_all.csv";
-} elsif ($nooi and ($cons eq 2)) { # CopraRNA 2 balanced prediction with overall consensus
-    system "cp CopraRNA2_final_balanced_consensus.csv CopraRNA_result.csv"; 
-    system "cp CopraRNA2_final_all_balanced_consensus.csv CopraRNA_result_all.csv";
-} elsif ($cons eq 1) { # CopraRNA 2 ooi prediction with ooi consensus
-    system "cp CopraRNA2_final_ooi_ooiconsensus.csv CopraRNA_result.csv";
-    system "cp CopraRNA2_final_all_ooi_ooiconsensus.csv CopraRNA_result_all.csv"; 
-} elsif ($cons eq 2) { # CopraRNA 2 ooi prediction with overall consensus
-    system "cp CopraRNA2_final_ooi_consensus.csv CopraRNA_result.csv";
-    system "cp CopraRNA2_final_all_ooi_consensus.csv CopraRNA_result_all.csv"; 
-} else { # CopraRNA 2 with org of interest focus (standard)
-    system "cp CopraRNA2_final_ooi.csv CopraRNA_result.csv";
-    system "cp CopraRNA2_final_all_ooi.csv CopraRNA_result_all.csv";
+    system "head -n $topcount CopraRNA_result_all.csv > CopraRNA_result.csv"; 
```

Comment:
@JensGeorg das taucht auch nochmal weiter unten in line 425 auf... ist das korrekt?